RFCs/0025-graph-data-model.md

```diff
@@ -54,51 +63,99 @@ Which is similarly extracted as variables:
 SELECT i, v FROM my_list AS v AT i
 ```
 
-For the graph data type, we model something very similar. A graph is a collection of *vertices* and *edges* that connect
-them with associated direction (which may be un-directed). All vertices and edges have a *label* (similar to the
-attribute name in struct) that must be a string or `NULL`. The value of a vertex or an edge can be any PartiQL value.
-The following diagram illustrates the model:
+For the graph data type, we model something very similar. A graph is a collection of *vertices* and *edges*, 
+where edges connect pairs of vertices and can be directed or undirected.
+Vertices and edges have *labels* (similar to the attribute names in struct). 
+There is also a **value at** each vertex and edge, which can be any PartiQL value.
```

Comment:
This is an interesting wording choice, I am interested to get your take on the distinction from the original text.